### ISSUE TITLE
CSV with repeated column

### ISSUE THREAD

**Issue by dlecina** [streamthoughts/kafka-connect-file-pulse#106]
Hello,
I have a CSV file with repeated columns, exported from Jira:
```
Issue Type,Issue key,Issue id,Summary,Assignee,Reporter,Priority,Status,Fix Version/s,Custom field (Epic Link),Resolution,Custom field (IT Project Name),Created,Updated,Due Date,Custom field (Defect Root Cause),Custom field (Fixed version),Sprint,Sprint,Sprint,Sprint,Sprint,Sprint,Sprint,Sprint,Sprint,Sprint,Sprint,Sprint,Sprint,Sprint,Sprint,Sprint,Custom field (Story Points),Labels
Bug,XXXX-1111,570394,FOO,BAR,BAZ,High,Available in UAT,Release 5.0,XXXX-2222,Resolved,FOO,3/Nov/20 12:17,28/Jan/21 17:11,,,,Sprint 2020 # 53,Sprint 2020 # 54,,,,,,,,,,,,,,,1.0,
```
This happens because Jira allows tickets to be included in multiple sprints, resulting in a repeating column in the case that one of the rows is included in as many sprints.

When trying to run it through the connector, I get the following error:
```io.streamthoughts.kafka.connect.filepulse.data.DataException: Cannot create field because of field name duplication Sprint```

Is there any way I can treat the repeated `Sprint` columns as an array, or otherwise work around the issue?

Thanks!

PS. I realize that there are other ways to ingest data from Jira. Unfortunately, I am faced with the requirement to import it as a CSV. I am also investigating XML, but it has its own set of challenges.

**Comment by dlecina** [comment]
I should also add that hardcoding the fields (such that I can rename each sprint column to something else) is not practical, as the number of Sprint columns will depend on which tickets are included in the Jira export, and thus change over time.

**Comment by fhussonnois** [comment]
Hi @dlecina, Yes currently the `DelimitedRowFilter` doesn't support repeated column. So the filter needs to be improved and unfortunately there is no easy way to deal with that using other FilePulse filters. I will work on a patch as soon as possible.

**Comment by dlecina** [comment]
Great to hear, thanks @fhussonnois. Ideally, I would expect that when a repeated column is found, it is interpreted as an array, but maybe it should be configurable in case someone wants to keep the error behavior.

**Comment by fhussonnois** [comment]
@dlecina I've pushed a new bug fix version 1.6.2. You can nown configure the property `duplicateColumnsAsArray=true` for the `DelimitedRowFilter`
